pkg/cli/admin/catalog/mirror.go

```diff
@@ -226,8 +227,30 @@ func (o *MirrorCatalogOptions) Complete(cmd *cobra.Command, args []string) error
 		if len(e.Mappings) < 1 {
 			return "", fmt.Errorf("couldn't extract database")
 		}
-		fmt.Fprintf(o.IOStreams.Out, "wrote database to %s\n", filepath.Join(e.Mappings[0].To, "bundles.db"))
-		return filepath.Join(e.Mappings[0].To, "bundles.db"), nil
+
+		fmt.Fprintf(o.IOStreams.Out, "wrote database to %s\n", e.Mappings[0].To)
+		var dbPath string
+		errFound := fmt.Errorf("found valid db file")
+		err := filepath.Walk(e.Mappings[0].To, func(path string, info os.FileInfo, err error) error {
+			if err != nil {
+				return nil
+			}
+			if info.IsDir() {
+				return nil
+			}
+			if _, err := sqlittle.Open(path); err == nil {
```

Comment:
Is it possible that more than one valid sqlite database file will be present there?

Reply:
Possible but unlikely (no tooling for building these images will do it) - but if that's the case, the output will indicate which one was picked, and if it's not the one you wanted you can override with `--path`